aksel.nav.no/website/components/website-modules/examples/__parts/PageDemoContent.tsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+
+function Content({ children }: { children: React.ReactNode }) {
+  return (
+    <div className="decorator-content">
+      <style>
+        {`.decorator-content {
+          width: 100%;
+          padding-block: 5rem;
+          padding-inline: 0.25rem;
+          height: 100%;
```

Comment:
I don't think width and height is needed
```suggestion
          padding-block: 5rem;
          padding-inline: 0.25rem;
```